src/index.ts

```diff
@@ -11,6 +11,7 @@ import * as reporter from './reporter'
 import * as automation from './automation'
 import * as shared from './shared'
 import * as apiClients from './api-clients'
+import { formatDateTime, formatDate, formatTime } from './formatting/dateTime'
```

Comment:
Can we simplify the signature here a bit?
```js
import { fmt } from '@cortezaproject/corteza-js'
fmt.time(...)
fmt.date(...)
fmt.dateTime(...)
```

I'm suggesting fmt mostly because it's a simple abbr. and a well-known package on the backend.